frontend/public/components/modals/add-secret-to-workload.tsx

```diff
@@ -163,7 +163,7 @@ export class AddSecretToWorkloadModal extends React.Component<AddSecretToWorkloa
     const addAsVolume = addAs === 'volume';
     const selectWorkloadPlaceholder = 'Select a workload';
 
-    return <form onSubmit={this.submit} name="co-add-secret-to-workload" className="co-add-secret-to-workload">
+    return <form onSubmit={this.submit} name="co-add-secret-to-workload" className="co-add-secret-to-workload modal-content">
```

Comment:
Rather than adding this class to every existing modal, I think it would be better to simply add a node to modal.tsx like so:

```
        <Modal
          isOpen={true}
          contentLabel="Modal"
          onRequestClose={closeModal}
          className="modal-dialog"
          overlayClassName="co-overlay"
          shouldCloseOnOverlayClick={!props.blocking}>
          <div className="modal-content">
            <Component {...props} cancel={closeModal} close={closeModal} />
          </div>
        </Modal>
```